### PR TITLE
fix nova resource

### LINKED ISSUE #59: Fees Demo

### PR DESCRIPTION
**Pull request by arwaawan3**
Closes #59 

This is not complete. I cannot figure out the locationBookingFees, locationProductFees and bookings relationships for the Fee model. I think a Fee has many locations, and locations have many bookings/products. But then there is a location_fees table that is confusing me. Would appreciate some clarity on how this works. Right now you can create/edit fees records but when you view it, there are errors with regards to those 3 fields (i.e. locationBookingFees, locationProductFees and bookings).
fees

### CODE REVIEW

**Comment by wolfrednicolas** [comment]
> mplete. I cannot figure out the locationBookingFees, locationProductFees and bookin

@arwaawan3 I checked as well and you right, neither `booking_fee_id` nor `product_fee_id` are in the migration for `fee`.. 

I think this is the approach ( almost completed)

https://github.com/tipoff/fees/pull/60/commits/d92565b4d791570b51bc2691998eb31db2dc3cbb

if not, revert the commit....

Also I improve some validations related to the rules 

https://github.com/tipoff/fees/pull/60/commits/7f2f18c4b94af031fabe348700b21368d62974a8

File: src/Nova/Fee.php [hunk 1]
```diff
@@ -70,8 +70,8 @@ public function fields(Request $request)
 
             new Panel('Data Fields', $this->dataFields()),
 
-            nova('location') ? HasMany::make('Location Booking Fee', 'locationBookingFees', nova('location')) : null,
-            nova('location') ? HasMany::make('Location Product Fee', 'locationProductFees', nova('location')) : null,
+            nova('location_fee') ? HasMany::make('Location Booking Fee', 'locationBookingFees', nova('location_fee')) : null,
+            nova('location_fee') ? HasMany::make('Location Product Fee', 'locationProductFees', nova('location_fee')) : null,
```
Comment (wolfrednicolas) [inline src/Nova/Fee.php:74]: @arwaawan3 this is the code I made is not done

**Comment by arwaawan3** [comment]
@wolfrednicolas Thanks for reviewing this! Your feedback really helps! 🙂

**Comment by joshtorres** [comment]
Thank you @arwaawan3 and @wolfrednicolas! Is this one ready?

**Comment by wolfrednicolas** [comment]
> Thank you @arwaawan3 and @wolfrednicolas! Is this one ready?

I think Is ready.

**Comment by arwaawan3** [comment]
@joshtorres Yes, but we need to make sure we also merge support package PR 147 to update nova config.

**Review by joshtorres — APPROVED** [review]
Thank you

**Pull request merged**